h/accounts/schemas.py

```diff
@@ -19,23 +20,20 @@
 log = logging.getLogger(__name__)
 
 PASSWORD_MIN_LENGTH = 2  # FIXME: this is ridiculous
-USERNAME_BLACKLIST = None
 
 
+@lru_cache(maxsize=None)
```

Comment:
According to the docs these should be marginally faster than maxsize=1 (removes the eviction logic)